sources/packages/backend/apps/api/src/route-controllers/application/application.students.controller.ts

```diff
@@ -624,6 +646,8 @@ export class ApplicationStudentsController extends BaseController {
         applicationOfferingChangeRequest?.applicationOfferingChangeRequestStatus;
     }
 
+    const assessmentTriggerType = application.currentAssessment.triggerType;
```

Comment:
please put a null check for current assessment as this API can be called during draft status as well. 

`const assessmentTriggerType = application.currentAssessment?.triggerType;`